### ISSUE TITLE
Dvorak's "dot" doesn't obey repeating setting

### ISSUE THREAD

**Issue by gergelypolonkai** [AnySoftKeyboard/AnySoftKeyboard#500]
When I press the dot key (".", above "e") and comma ("," above "o") on twe Dvorak set twice, they insert ">" and "<” respectively, even if I disable key repeating. More strange is, they don't have these characters on their long press popup (they don't have long press popup at all).

**Comment by Soundtoxin** [comment]
Yes! This is killing me. I can't type three dots in a row unless I take very long pauses.

**Comment by ArenaL5** [comment]
This happens because long press popup characters are completely separate from multi-tap characters.

Long press popup characters appear when you hold down a key. Multi-tap characters appear when you rapidly tap a key more than once. Still, having multi-tap characters on the dot key looks like bad design to me.

You can fix your problem by using a different setting instead of "key repeating": try UI → Tweaks and more → Tweaks → Multi-tap timeout (under Behavior tweaks).

**Comment by Soundtoxin** [comment]
Even on the fastest timeout setting I have to consciously wait quite a bit between presses. Thanks for the tip, though. I had not yet tweaked that, so it should be marginally better now.

**Comment by ArenaL5** [comment]
Above the fastest timeout there's an option to disable multi-tap altogether (it just reads "disabled" in English). Isn't that working properly?

**Comment by Soundtoxin** [comment]
That works great! I should've tried that. Thanks again.
